src/sentry/api/endpoints/org_auth_tokens.py

```diff
@@ -62,18 +66,30 @@ def post(
         organization_context: RpcUserOrganizationContext,
         organization: RpcOrganization,
     ) -> Response:
-        token_str = generate_token(organization.slug, generate_region_url())
+        try:
+            token_str = generate_token(organization.slug, generate_region_url())
+        except SystemUrlPrefixMissingException:
+            return Response(
+                {
+                    "detail": {
+                        "message": "system.url-prefix is not set. You need to set this to generate a token.",
+                        "code": "missing_system_url_prefix",
+                    }
+                },
+                status=400,
+            )
```

Comment:
Most of our API errors don't have nested structures, and `detail` is generally a string or a list of strings. This could make typing error responses consistently more challenging in the future. We could have `detail` and `code` as sibling properties in the response.

Reply:
I modeled this based on the error response type in the frontend, which is:

```ts
type ResponseJSON = {
  [key: string]: unknown;
  detail?: string | {code?: string; message?: string};
};
```

(It's not used a lot, but it exists 😅 )

but happy to use a different schema if preferred!

Reply:
I didn't know we had other nested structures already. In that case my feedback isn't relevant.